File: PHP/Car.php
<?php
class Car {

    public $id;
    public $license;
    public $driver;
    private $passengenger;

    public function __construct($lic, $dri) {
        $this->license = $lic;
        $this->driver = $dri;
        $this->passengenger = null;
    }

    public function printDataCar() {
        echo "License Car: {$this->license} and model is: {$this->model} pasenger: {$this->passengenger}<br>";
    }

    public function getPasenger() {
        return $this->passengenger;
    }

    public function setPasenger($passenger) {
        if($passenger == 4){
            $this->passengenger = $passenger;
        } else {
            echo "El valor de Passenger: debe ser cuatro.<br>";
        }
    }
}

File: Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("", "")
    passegenger = int
    def __init__(self,license, driver):
        self.license = license
        self.driver = driver
    def getPasengers(self):
        return self.passegenger
    def setPasengers(self, pasenger):
        if(pasenger == 4):
            self.passegenger = pasenger
        else:
            print("passeger debe ser igual a cuatro!")
